P/porta/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "porta"
+version = v"1.4.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/bdoolittle/julia-porta.git", "93b99656f390c2252a0142bf9a4c22f5217a42da")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir ../destdir/bin
+
+if [[ $target == x86_64-w64-mingw32 ]]; then
+    make -C julia-porta/gnu-make/ CC=x86_64-w64-mingw32-gcc;
+    cp julia-porta/gnu-make/bin/xporta.exe ../destdir/bin/xporta.exe;
+    cp julia-porta/gnu-make/bin/valid.exe ../destdir/bin/valid.exe;
+elif [[ $target == i686-w64-mingw32 ]]; then
+    make -C julia-porta/gnu-make/ CC=i686-w64-mingw32-gcc;
+    cp julia-porta/gnu-make/bin/xporta.exe ../destdir/bin/xporta.exe;
+    cp julia-porta/gnu-make/bin/valid.exe ../destdir/bin/valid.exe;
+else
+    make -C julia-porta/gnu-make/;
+    cp julia-porta/gnu-make/bin/xporta ../destdir/bin/xporta;
+    cp julia-porta/gnu-make/bin/valid ../destdir/bin/valid;
+fi
```

Comment:
We have some [environment variables](https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Automatic-environment-variables-1) predefined in the build environment that will make much simpler:
```suggestion
mkdir -p ${bindir}
make -C julia-porta/gnu-make/ CC=${CC}
cp julia-porta/gnu-make/bin/xporta${exext} ${bindir}
cp julia-porta/gnu-make/bin/valid${exext} ${bindir}
```

Reply:
Hey, thanks for the tip! The script is much cleaner with the environment variables. I'll keep this in mind in the future.

I corrected the changes in a separate commit  as the suggestion dropped an 'e' in `${exeext}`.

Reply:
Good catch!  I always miss the extra "e" :sweat_smile: